styleguide/derek/_partials/_basement.ejs

```diff
@@ -1,17 +1,15 @@
 <!--[basement]-->
 <div class="bg-dark-gray">
-  <div class="container basement">
+  <div class="basement">
     <div class="row">
```

Comment:
I created a `row` class that simply includes the mixin `make-row`
I'd rather not have duplicate css for rows per module but if we would like to take that approach I'd prefer we have a row placeholder class so that the sass will output multiple classes with the same row styling in one place.